frontend/src/components/GamemodeBar.jsx

```diff
@@ -23,74 +23,79 @@ function GamemodeBar() {
     }
 
     return (
-      <div className="gamemode-bar">
+      <>
+      
+      {gameState === 'pregame' ? (
```

Comment:
unlike the comment above, states like these are fine inside the components since they are component specific and don't determine whether or not the component renders